### PR TITLE
[ZH] Update `Becoming a Beatmap Nominator`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md [hunk 1]
```diff
@@ -53,19 +90,23 @@ BN 申请程序于 [NAT 官网](http://bn.mappersguild.com/bnapps)进行。
...
 
-### 在离开后重回 BN 团队
+如果申请者申请失败，但是申请者几乎快达到了通过条件时，冷却期限可能会根据情况缩短。
 
-对于玩家，在离开后重回 BN 团队的要求，与第一次加入的要求不太一样。如果玩家在离开团队之前表现良好，他们在重回团队时的要求可能会适当放宽，甚至可以无要求直接加入。NAT 会提示那些能够快速返回 BN 团队的玩家。
+### 在离开 BN 团队后重新加入
+
+对于离开 BN 团队后重新加入玩家，和第一次加入的玩家的要求不太一样。如果玩家在离开团队之前表现良好，他们在重回团队时的要求可能会适当放宽，甚至可以无要求直接加入。NAT 会提示那些能够快速返回 BN 团队的玩家。
 
 如果你曾是 BN，且你未能通过 BN 申请，则需要满足这些要求，以再次提交申请。
```
Comment (Sunnyboy971) [inline wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md:99]: This line is removed in the latest known version of `en.md`.

**Review by Sunnyboy971 — CHANGES_REQUESTED** [review]

File: wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md [hunk 2]
```diff
@@ -1,47 +1,84 @@
...
-以下这些是对提交摸的特殊要求。任何一条不满足都可能导致 BN 申请被拒。
+以下这些是对提交摸的特殊要求。任何一条不满足都可能导致申请者被拒。
 
 - **覆盖所有难度等级。**
+  - 大多数图拥有从简单 (Easy) 到专家 (Expert) 的一系列[难度](/wiki/Beatmap/Difficulty)，所以 BN 也应能够熟练地摸这些难度。对于摸非常困难难度的能力不做要求。
 - **识别出摸图工具无法显示出的，不符合上架标准 (unrankable) 的问题。如不准确的歌曲时间轴或是歌曲元数据问题。**
 - **比较谱面不同段落，并提出问题或改进建议。**
-- **评述谱面的整体元素。如节奏、物件间距、物件运动、物件密度、不同段落的差异、谱面一致性。**
+  - 引用谱面的不同段落，能让提出的建议和谱师在谱面中展示出的原思路、风格保持一致。
+- **评述谱面的整体元素。如[节奏](/wiki/Music_theory/Rhythm)、物件间距、物件运动、物件密度、不同段落的差异、谱面一致性。**
```
[the remark this answers is not in the document]
Comment (YumeMuzi) [reply wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md:30]: no, the technical term `movement` also included how the cursor turn, accelerates, decelerates, etc..., and how the patterns or sliders trend too. It's not simply the distance between objects.

[the remark this answers is not in the document]
Comment (YumeMuzi) [reply wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md:30]: 而`移动`则描述得比较浅显，`运动`则概括了它们。

File: wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md (hunk 1 above)
Comment (YumeMuzi) [reply wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md:99]: I forgot to delete the line above

**Review by Sunnyboy971 — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**